api/node/index.ts

```diff
@@ -930,17 +930,30 @@ var globalEventLoop: EventLoop = new EventLoop;
  * If the event loop is already running, then this function returns the same promise as from
  * the earlier invocation.
  *
- * @param runningCallback Optional callback that's invoked once when the event loop is running.
+ * @param args As Function it defines a callback that's invoked once when the event loop is running.
+ * @param args.runningCallback Optional callback that's invoked once when the event loop is running.
  *                         The function's return value is ignored.
+ * @param args.quitOnLastWindowClosed if set to `true` event loop is quit after last window is closed otherwise
+ *                          it is closed after {@link quitEventLoop} is called.
+ *                          This is useful for system tray applications where the application needs to stay alive even if no windows are visible.
+ *                          (default true).
```

Comment:
The default should be true, yes, but in the implementation it isn't: It defaults to false. If you call runEventLoop(), the field will be undefined, and that will evaluate to false in line 908.

I think that you need to check that if the field is not specified, you _do_ implement the default of true, and change the signature of `start()` to not take an _optional_ boolean but require the caller to have decided by then - that way you can isolate the fallback handling to runEventLoop.

Reply:
changed